theme/js/toc.js

```diff
@@ -74,5 +74,35 @@
     }
 
     render[settings.showEffect]();
+
+    var tocLinkItems = $('#current > a.toc__link-item'); // Get all the TOC header links
+    var activeLink = null; // Store the currently active TOC link
+
+    $(window).on('scroll', function() {
```

Comment:
I think this will need to handle scroll up and down differently. For example, if I go to /archivematica-local/appendix/ and scroll all the way down, the last header gets highlighted. Then, as I scroll up and that header and the content under it is not longer visible, that header is still highlighted until the previous header is visible. Given that the content I am seeing is from that previous header it's a little confusing...hope that makes sense